integration_tests/run_pyspark_from_build.sh

```diff
@@ -85,14 +85,20 @@ else
     RUN_DIR="$SCRIPTPATH"/target/run_dir
     mkdir -p "$RUN_DIR"
     cd "$RUN_DIR"
+
+    export PYSP_TEST_spark_driver_extraJavaOptions="-ea -Duser.timezone=UTC $COVERAGE_SUBMIT_FLAGS"
+    export PYSP_TEST_spark_driver_extraClassPath="${ALL_JARS// /:}"
```

Comment:
I think when running with spark-submit configs that impact the driver command line arguments have to be set on the command line of spark-submit because the driver is launched first and it launches the python process.  When using find-spark that is not the case. 

Did you test these changes with `TEST_PARALLEL=1`?

Reply:
Double-checked with the command in the description for `TEST_PARALLEL=1` and `TEST_PARALLEL=2`